apps/web/src/components/molecules/ComingEvent/ComingEvent.tsx

```diff
@@ -14,28 +15,32 @@ interface ComingEventProps {
 
 // TODO: mye relative og absolute her.... too bad!
 export const ComingEvent: React.FC<ComingEventProps> = (props) => (
-  <div className="relative h-[100px] max-w-[400px] overflow-hidden rounded-xl shadow-md">
-    <div className="m-0 inline-block align-top">
-      <Image src={props.img} alt={props.title} width={100} height={100} className="object-cover" />
+  <Link href={props.info_link}>
+    <div className="mt-2 flex flex-col">
+      <div className="relative">
+        <Image
+          src={props.img}
+          alt={props.title}
+          width={320}
+          height={180}
+          className="rounded-xl border-2 border-slate-4 object-cover"
+        />
+        <Badge color="green" variant="solid" className="absolute bottom-2 left-2">
+          {props.tag}
+        </Badge>
+      </div>
+      <div className="">
+        <span>
+          <span>{/* PLACEHOLDER */}</span>
+          <div>
+            <p className="m-0">{props.title}</p>
+            <p className="m-0">{props.date}</p>
```

Comment:
I think datetime-formatting should be determined here?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat

Reply:
ComingEvent trenger en del arbeid ja